doc/POST_UPGRADE.d/4.1.5~ynh3.md

```diff
@@ -1,4 +1,4 @@
 I Hate Money was successfully upgraded :)
-Please open https://__DOMAIN____PATH_URL__/admin
+Please open https://__DOMAIN____PATH__/admin
 The admin password is: __PASSWORD__
-If you facing an issue or want to improve this app, please open a new issue in this project: https://github.com/YunoHost-Apps/ihatemoney_ynh
\ No newline at end of file
+If you facing an issue or want to improve this app, please open a new issue in this project: https://github.com/YunoHost-Apps/ihatemoney_ynh
```

Comment:
Hmmm not sure to understand why this is specific to version 4.1.5 ? And also why would you need to display the admin password when upgrading o_O (Maybe there's a good reason but sounds pretty funky)

Reply:
git log reveals : 

> +* During upgrade from version 4.1.5~ynh3, a new admin password is generated and sent to the root